src/App.jsx

```diff
@@ -14,70 +16,64 @@ export const goods = [
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood ? (
+          `${selectedGood} is selected`) : (
+          'No goods selected')}
+        {selectedGood && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setSelectedGood('')}
+          />
+        )}
+      </h1>
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              data-cy="Good"
+              key={good}
+              className={classNames({
+                'has-background-success-light': selectedGood === good,
+              })}
             >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
-
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Garlic
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </main>
-);
+              <td>
+                <button
+                  data-cy={selectedGood === good
+                    ? 'RemoveButton'
+                    : 'AddButton'
+              }
+                  type="button"
+                  className={classNames('button',
+                    { 'is-info': selectedGood === good })}
+                  onClick={() => setSelectedGood(
+                    selectedGood === good
+                      ? ''
+                      : good,
+                  )}
+                >
+                  {selectedGood === good
+                    ? '-'
+                    : '+'
+                }
+                </button>
```

Comment:
if you change almost every attr depending on the selectedGood === good condition, better to write
```
{selectedGood === good 
 ? (<button></button>)
 : (<button><button/>)}
```

Reply:
what do you think?

Reply:
smth like this (commit 2)